$OpenBSD: patch-m4_as-host-defines_m4,v 1.1.1.1 2010/03/25 10:23:26 sthen Exp $

orc's arm support is broken; avoid testing for this so the
generic code is used instead.

--- m4/as-host-defines.m4.orig	Thu Mar 25 09:39:08 2010
+++ m4/as-host-defines.m4	Thu Mar 25 09:40:25 2010
@@ -23,10 +23,10 @@ AC_DEFUN([AS_HOST_DEFINES],
       HAVE_POWERPC=yes
       AC_DEFINE(HAVE_POWERPC, 1, [Defined if host is powerpc])
       ;;
-    xarm)
-      HAVE_ARM=yes
-      AC_DEFINE(HAVE_ARM, 1, [Defined if host is arm])
-      ;;
+dnl    xarm)
+dnl      HAVE_ARM=yes
+dnl      AC_DEFINE(HAVE_ARM, 1, [Defined if host is arm])
+dnl      ;;
   esac
 
 AM_CONDITIONAL(HAVE_I386, test "x$HAVE_I386" = "xyes")
